website/pages/docs/block.mdx

```diff
@@ -6,6 +6,22 @@ description: 'Million.js introduces the concept of a Block. Blocks are a way for
 import { Callout } from 'nextra-theme-docs';
 import { AutomaticModeWarning } from '../../components/automatic-mode-warning';
 
+## What's a block?
+
+Million.js is a library that enables you to create _blocks_. A block is a special [Higher Order Component (HOC)](https://legacy.reactjs.org/docs/higher-order-components.html) that can be used as a React component but is hyper-optimized for rendering speed.
+
+Blocks are essentially components wrapped by `block(){:jsx}`.
```

Comment:
I try to avoid "can" or words of the like if things are concrete. So if blocks *are* just a component wrapped by `block(){:jsx}` then you can say that, unless there are edge cases. In which case, you should elabroate those.

```suggestion
Blocks are components wrapped by `block(){:jsx}`.
```

Reply:
Thanks, Sarah!!! 🙌!! You're so experienced. Damn!!